.profile.d/heroku-metrics-daemon.sh

```diff
@@ -10,6 +10,24 @@ export HEROKU_METRICS_PROM_PORT=$(expr $PORT + 1)
 export HEROKU_PROM_METRICS_ENDPOINT=${HEROKU_METRICS_PROM_ENDPOINT}
 export HEROKU_PROM_METRICS_PORT=${HEROKU_METRICS_PROM_PORT}
 
+BUILD_DIR=/app
+DOWNLOAD_URL=$(curl --retry 3 -s https://api.github.com/repos/heroku/agentmon/releases/latest | grep "browser_download_url" | awk -F': ' '{print $2}' | sed -e 's/"//g')
```

Comment:
This is good. I would like to provision an S3 bucket for this to guard against github issues down the road. Yes, I know S3 went down recently, but if s3 is down, so are builds because they store cache/slugs to S3.